be1-go/channel/chirp/mod.go

```diff
@@ -348,8 +343,7 @@ func (c *Channel) verifyAddChirpMessage(msg message.Message) error {
 	return nil
 }
 
-func (c *Channel) verifyDeleteChirpMessage(msg message.Message) error {
-	var chirpMsg messagedata.ChirpDelete
+func (c *Channel) verifyDeleteChirpMessage(msg message.Message, chirpMsg messagedata.ChirpDelete) error {
```

Comment:
Why did you not use your interface messagedata.VerifiableMessageData here ?

Reply:
I hesitated quite a bit but ultimately decided against it. On the plus side, it would remove a few duplicated line of codes. On the other hand I think it would reduce code lisibility, since in the end the verify functions for add and delete do not do exactly the same. So either add other verify functions with one doing everything that is common for both and 2 that do the rest. I was not enthusiastic since there are already quite a lot of verify functions. The other solution is to add if - else in one common verify function but it adds code complexity. 

All in all, it seemed better to let things separate even though a few duplication is the price.